docs/S00-intro/L4-technical-requirements/index.md

```diff
@@ -90,7 +91,7 @@ Even if JavaScript is not your first software language, it's really important yo
 *   [Markdown All in One](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one){target=_blank} All you need to write Markdown
 *   [GitHub Linker](https://marketplace.visualstudio.com/items?itemName=gimenete.github-linker){target=_blank} Create links to fragments of code in GitHub
 
-We strongly recommend using VSCode in this course, particularly if you're new to coding, due to its extensive plugins. Other popular editors include [Atom,](https://atom.io/){target=_blank} [Sublime Text,](https://www.sublimetext.com/){target=_blank} and [IDEA](https://www.jetbrains.com/idea/){target=_blank} (written originally for Java but now supports many languages).
```

Comment:
I'm still so upset about Atom being sunsetted. RIP 🪦.

Reply:
Me too! But now website is live again. When I checked atom.io was not even working. Abandonware is also cool.